apps/bmd/src/pages/AdminScreen.tsx

```diff
@@ -61,6 +59,12 @@ const AdminScreen = ({
     updateAppPrecinctId(event.currentTarget.value)
   }
 
+  const [isFetchingElection, setIsFetchingElection] = useState(false)
+  const loadElection = () => {
+    setIsFetchingElection(true)
+    fetchElection()
+  }
```

Comment:
This setup means we'll never set it to `false` again. Though that may work, it feels weird.

Reply:
If we wanted to set it back to false I think you could add a useEffect that depends on election and set isFetchingElection back to false once that prop is populated.

Reply:
Yes, we could. 

I was thinking through the logic and realized that the ability to set this state back to false didn't matter because once is an `election` is loaded the logic follows a separate path. 

Additionally, if we add this logic, then we have to test it and have a path to get into this state. I'm drawn towards simpler logic when possible.

It is possible that loading the election from the Admin card could fail. In the _rare_ case that this happens, the Admin can remove and re-insert the card.